obspy/io/stationtxt/core.py

```diff
@@ -11,21 +11,24 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-from future.builtins import *  # NOQA
-from future.utils import native_str
-from future import standard_library
-with standard_library.hooks():
-    import collections
-
 import csv
 import io
+import re
 import warnings
+import six
```

Comment:
`six` is not an ObsPy dependency. Please remove it and either use functionality from the `future` module or `obspy.core.compatibility`.